examples/custom_image_embedding_search.ipynb

```diff
@@ -0,0 +1,484 @@
/4wQ9w5ZVX6oQ7AwMDePzxx/XxjdpXNhzp/8d//Afuuusu3Hvvvfi///s/XHXVVbjpppswMjKy3kVbU1xxxRUYHBzUn2eeeUYf++pXv4r/+Z//wU9/+lMcOHAAZ8+exa233rqOpV0d5HI5XHXVVXjggQeMx++//3780z/9E374wx/ihRdeQCgUwk033YRisajP+fSnP43XX38dTzzxBB577DH85je/wRe/+MW1qsKqoFq7AMDNN9/s6D8//vGPHccvxXY5cOAA9u7di+effx5PPPEESqUS9uzZg1wup8+pNnbm5uZwyy23YGZmBs899xz+/d//HQ899BC+/e1vr0eVVgSLaRcA+MIXvuDoM/fff78+dim2S1dXF77//e/jpZdewosvvogPf/jD+OhHP4rXX38dwAbuK2qD4f3vf7/au3ev/n9ubk51dHSoffv2rWOp1hb33nuvuuqqq4zHJicnVUNDg/rpT3+qf3vzzTcVAHXw4ME1KuHaA4B65JFH9P/lclmlUin1//7f/9O/TU5OqsbGRvXjH/9YKaXUG2+8oQCo//3f/9XnPP7448rj8agzZ86sWdlXE7JdlFLq9ttvVx/96Eddr6mFdlFKqZGREQVAHThwQCm1uLHzi1/8Qnm9XjU0NKTP+cEPfqCi0aianp5e2wqsEmS7KKXUhz70IfUXf/EXrtfUQrsopVQ8Hlf/+q//uqH7yobS9GdmZvDSSy9h9+7d+jev14vdu3fj4MGD61iytcc777yDjo4ObNq0CZ/+9Kdx8uRJAMBLL72EUqnkaKPt27ejp6enptro2LFjGBoacrRDLBbDddddp9vh4MGDaGpqwvve9z59zu7du+H1evHCCy+seZnXEvv370dbWxu2bduGO+64A+Pj4/pYrbTL1NQUAKC5uRnA4sbOwYMHsWvXLiSTSX3OTTfdhHQ6rTXAjQ7ZLoQf/ehHSCQS2LlzJ+655x7k83l97FJvl7m5OfzkJz9BLpfDwMDAhu4rG2rDnbGxMczNzTkaEQCSySTeeuutdSrV2uO6667DQw89hG3btmFwcBD33Xcffv/3fx+vvfYahoaG4PP50NTU5LgmmUxiaGhofQq8DqC6mvoKHRsaGkJbW5vjeH19PZqbmy/ptrr55ptx6623or+/H0ePHsU3v/lNfOQjH8HBgwdRV1dXE+1SLpfxla98BR/84Aexc+dOAFjU2BkaGjL2KTq20WFqFwD41Kc+hd7eXnR0dODQoUO4++67cfjwYfzsZz8DcOm2y6uvvoqBgQEUi0WEw2E88sgj2LFjB1555ZUN21c2FOlbnMdHPvIR/feVV16J6667Dr29vfjP//xPBAKBdSyZxUbAH//xH+u/d+3ahSuvvBKbN2/G/v37ceONN65jydYOe/fuxWuvveaIhbFwbxcez7Fr1y60t7fjxhtvxNGjR7F58+a1LuaaYdu2bXjllVcwNTWF//qv/8Ltt9+OAwcOrHexLggbyryfSCRQV1e3IEJyeHgYqVRqnUq1/mhqasJll12GI0eOIJVKYWZmBpOTk45zaq2NqK6V+koqlVoQADo7O4tz587VVFtt2rQJiUQCR44cAXDpt8udd96Jxx57DE8//TS6urr074sZO6lUytin6NhGhlu7mHDdddcBgKPPXIrt4vP5sGXLFlxzzTXYt28frrrqKvzjP/7jhu4rG4r0fT4frrnmGjz55JP6t3K5jCeffBIDAwPrWLL1RTabxdGjR9He3o5rrrkGDQ0NjjY6fPgwTp48WVNt1N/fj1Qq5WiHdDqNF154QbfDwMAAJicn8dJLL+lznnrqKZTLZT2p1QJOnz6N8fFxtLe3A7h020UphTvvvBOPPPIInnrqKfT39zuOL2bsDAwM4NVXX3UIRU888QSi0Sh27NixNhVZYVRrFxNeeeUVAHD0mUutXUwol8uYnp7e2H1l3UIIl4mf/OQnqrGxUT300EPqjTfeUF/84hdVU1OTI0LyUsfXvvY1tX//fnXs2DH17LPPqt27d6tEIqFGRkaUUkp96UtfUj09Peqpp55SL774ohoYGFADAwPrXOqVRyaTUS+//LJ6+eWXFQD1d3/3d+rll19WJ06cUEop9f3vf181NTWpRx99VB06dEh99KMfVf39/apQKOh73Hzzzeo973mPeuGFF9Qzzzyjtm7dqj75yU+uV5VWBJXaJZPJqL/8y79UBw8eVMeOHVO//vWv1Xvf+161detWVSwW9T0uxXa54447VCwWU/v371eDg4P6k8/n9TnVxs7s7KzauXOn2rNnj3rllVfUL3/5S9Xa2qruueee9ajSiqBauxw5ckR997vfVS+++KI6duyYevTRR9WmTZvU9ddfr+9xKbbLN77xDXXgwAF17NgxdejQIfWNb3xDeTwe9atf/UoptXH7yoYjfaWU+ud//mfV09OjfD6fev/736+ef/759S7SmuITn/iEam9vVz6fT3V2dqpPfOIT6siRI/p4oVBQf/7nf67i8bgKBoPq4x//uBocHFzHEq8Onn76aQVgwef2229XSp1ftvdXf/VXKplMqsbGRnXjjTeqw4cPO+4xPj6uPvnJT6pwOKyi0aj60z/9U5XJZNahNiuHSu2Sz+fVnj17VGtrq2poaFC9vb3qC1/4wgKh+VJsF1ObAFAPPvigPmcxY+f48ePqIx/5iAoEAiqRSKivfe1rqlQqrXFtVg7V2uXkyZPq+uuvV83NzaqxsVFt2bJFff3rX1dTU1OO+1xq7fJnf/Znqre3V/l8PtXa2qpuvPFGTfhKbdy+YrfWtbCwsLCwqBFsKJ++hYWFhYWFxfJhSd/CwsLCwqJGYEnfwsLCwsKiRmBJ38LCwsLCokZgSd/CwsLCwqJGYEnfwsLCwsKiRmBJ38LCwsLCokZgSd/CwsLCwqJGYEnfwsLCwsKiRmBJ38LCwsLCokZgSd/CwsLCwqJGYEnfwsLCwsKiRvD/AQGIBm//Wc2PAAAAAElFTkSuQmCC",
```

Comment:
put image in images folder, add reference here (examples in [assistants cookbook](https://cookbook.openai.com/examples/assistants_api_overview_python))